src/main/java/pro/sky/JavaCoredemo2/controller/MathQuestionController.java

```diff
@@ -0,0 +1,35 @@
+package pro.sky.JavaCoredemo2.controller;
+
+import pro.sky.JavaCoredemo2.entity.Question;
+import pro.sky.JavaCoredemo2.services.MathQuestionService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+@RestController
+@RequestMapping("/exam/math")
+public class MathQuestionController {
+
+    private final MathQuestionService service;
+
+    public MathQuestionController(MathQuestionService service) {
```

Comment:
```suggestion
    public MathQuestionController(QuestionService service) {
```
инъекцию надо всегда делать по интерфейсу, а не реализации

Reply:
также см. внимательнее задание про Qualifier аннотацию